R/geom-raster.R

```diff
@@ -90,7 +90,7 @@ GeomRaster <- ggproto("GeomRaster", Geom,
 
   draw_panel = function(self, data, panel_params, coord, interpolate = FALSE,
                         hjust = 0.5, vjust = 0.5) {
-    if (!inherits(coord, "CoordCartesian")) {
+    if (!inherits(coord, "CoordCartesian") || !coord$is_linear()) {
```

Comment:
Shouldn't it be enough to test `coord$is_linear()` and forget about the special case for `CoordCartesian`?

The message below also needs an update

Reply:
Yeah good points, I agree